src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-container.component.ts

```diff
@@ -309,6 +309,17 @@ export class CodeEditorTutorAssessmentContainerComponent implements OnInit, OnDe
     readOnly() {
         return !this.isAtLeastInstructor && !!this.complaint && this.isAssessor;
     }
+    
+    /**
+     * Removes the username from the repositoryURL
+     */
+    adjustedRepositoryURL(): string {
+        if (this.participation.repositoryUrl != null) {
+           const splitted = this.participation.repositoryUrl.split('@', 2);
+            return 'http://' + splitted[1];
```

Comment:
We should not hardcode `http://` here. It would be better to extract the schema from the repository url (if possible).

Reply:
Should now be fixed 👍